### PR TITLE
docs: add information about permissions for azure pat

### PR DESCRIPTION
**Pull request by danielkoch**
## Changes
Added needed permissions for Azure PAT

## Context
Extend documentation while working with renovate on Azure.

## Documentation

- [x] I have updated the documentation, or
- [ ] No documentation update is required

## How I've tested my work

I have verified these changes via:

- [x] Code inspection only, or
- [ ] Newly added/modified unit tests, or
- [ ] No unit tests but ran on a real repository, or
- [ ] Both unit tests + ran on a real repository

### CODE REVIEW

**Review by viceice — COMMENTED** [review]
Workitem scope is only needed for link to workitem

**Comment by danielkoch** [comment]
@viceice: Right, I added a note.

File: lib/modules/platform/azure/readme.md [hunk 1]
```diff
@@ -9,6 +9,13 @@ Let Renovate use your PAT by doing _one_ of the following:
 - Set your PAT as an environment variable `RENOVATE_TOKEN`
 - Set your PAT when you run Renovate in the CLI with `--token=`
 
+Permissions for your PAT should be at minimum:
+
+| Scope            Permission                                       |
+| -------------- | -------------------------------------------------|
+| [`Work Items`] | Read & write (only needed for link to work item) |
```
Comment (viceice) [inline lib/modules/platform/azure/readme.md:16]: please add a description column to the table

**Review by rarkins — DISMISSED** [review]

**Review by rarkins — APPROVED** [review]

**Review by viceice — APPROVED** [review]

**Comment by viceice** [comment]
cla not signed

**Pull request merged**